static/_redirects

```diff
@@ -497,3 +497,4 @@
 /docs/setup/cluster-large/     /docs/setup/best-practices/cluster-large/   301
 /docs/setup/node-conformance/     /docs/setup/best-practices/node-conformance/   301
 /docs/setup/certificates/     /docs/setup/best-practices/certificates/   301
+/concepts/configuration/resource-bin-packages/   /concepts/scheduling-eviction/resource-bin-packages/ 301
```

Comment:
```suggestion
/concepts/configuration/resource-bin-packing/   /concepts/scheduling-eviction/resource-bin-packing/ 301
```

Reply:
/assign
hi @sftim does this need a technical review or is this just moving things around?  Let me know and I will line up someone from SIG scheduling to do a technical review. Thanks

Reply:
This is just a rename.